### PR TITLE
loader for single parquet file

### PR DESCRIPTION
**Pull request by sarakolding**

### CODE REVIEW

File: psycop/common/model_training_v2/trainer/data/test_dataloaders.py [hunk 1]
```diff
@@ -7,13 +7,30 @@
...
+    df = pl.DataFrame({"a": [1, 2, 3]})
+
+    parquet_path = Path(tmpdir) / "test.parquet"
+
+    df.write_parquet(parquet_path)
+
+    parquet = ParquetLoader(str(parquet_path)).load().collect()
+
+    assert len(parquet) == len(df)
+    assert parquet.columns == df.columns
```
Comment (MartinBernstorff) [inline psycop/common/model_training_v2/trainer/data/test_dataloaders.py:28]: Fint bare at tjekke at filen kan loades; resten er Polars funktionalitet. Men en anelse overkill har ikke skadet os 😉

**Review by MartinBernstorff — APPROVED** [review]
Ser super ud! Småting, så er den klar.

**Comment by MartinBernstorff** [comment]
@HLasse Have removed you as reviewer, but I imagine an FYI-style review is OK here 👍

**Comment by MartinBernstorff** [comment]
@sarakolding We might even want to merge the loaders, so we can load both a single file or multiple files from the same loader.

File: psycop/common/model_training_v2/trainer/data/dataloaders.py [hunk 2]
```diff
@@ -12,9 +12,42 @@ class MissingPathError(Exception):
...
+            paths: Path to parquet file.
+            validate_on_init: Whether to validate the path on init.
+                Helpful when testing the .cfg parses, where the absolute path will differ between devcontainer and Ovartaci.
+                Defaults to True.
+
+        """
+        self.dataset_paths = Path(paths)
+
+        if validate_on_init:
+            missing_path = self._check_path_exists(self.dataset_paths)
```
[the remark this answers is not in the document]
Comment (sarakolding) [reply psycop/common/model_training_v2/trainer/data/dataloaders.py:30]: tror ikke jeg forstår?

**Comment by sarakolding** [comment]
> @sarakolding We might even want to merge the loaders, so we can load both a single file or multiple files from the same loader.

is it a request or a suggestion, @MartinBernstorff ? 👀

**Comment by sarakolding** [comment]
> @sarakolding We might even want to merge the loaders, so we can load both a single file or multiple files from the same loader.

i'm an eejit, you can just load a single file using the vertical concatenator

**Comment by sarakolding** [comment]
@MartinBernstorff ok to just close?

**Comment by MartinBernstorff** [comment]
Oh, I feel like we tried that! How did you come about loading a single file with the vertical concatenator?

**Comment by MartinBernstorff** [comment]
Always OK to close [fire](https://app.graphite.dev/macros/Aarhus-Psychiatry-Research)

**Comment by sarakolding** [comment]
> Oh, I feel like we tried that! How did you come about loading a single file with the vertical concatenator?

i also thought i tried that. i think i just got a bunch of errors in the beginning, and must have assumed that it was because of the loader. i don't remember exactly what the error message was, but i think it was a bit vague.

**Pull request closed without merging**